### PR TITLE
update version numbers on yt-4.0 branch

### PR DESCRIPTION
**Pull request by ngoldbaum**
This PR tests to make sure jenkins isn't doing silly things on the yt-4.0 branch. It also updates the version number to reflect the planned future version based on this branch.

### CODE REVIEW

**Comment by ngoldbaum** [comment]
Hmm, i wonder why travis and appveyor aren't building cykdtree. That was working in this branch before when I was using my repo...

**Comment by ngoldbaum** [comment]
I'm gonna try to reproduce this final windows test failure on my windows machine at home.

File: appveyor.yml [hunk 1]
```diff
@@ -30,6 +29,9 @@ install:
 
     # Install specified version of numpy and dependencies
     - "conda install --yes -c conda-forge numpy scipy nose setuptools ipython Cython sympy fastcache h5py matplotlib flake8 mock"
+    # HACK, FIXME BEFORE MERGING
```
Comment (jzuhone) [inline appveyor.yml]: @ngoldbaum didn't you want to merge this?

Comment (ngoldbaum) [reply appveyor.yml]: That’s a note to fix that issue before merging with master. We need to do a proper cykdtree release.

**Pull request merged**